packages/skeleton/src/skeleton.tsx

```diff
@@ -198,12 +205,17 @@ const AnimatedGradient = React.memo(
     measuredWidth,
     colors,
     transition = {},
+    animationDirection,
   }: {
     measuredWidth: number
     colors: string[]
     transition?: MotiTransitionProp
+    animationDirection?: 'left-to-right' | 'right-to-left'
   }) {
     const backgroundSize = 6
+    const reversed = animationDirection === 'left-to-right'
+    const fromPosition = { translateX: 0 }
```

Comment:
Maybe we should use left position and right position here. `to` and `from` should be names of variables _after_ checking the animation direction.

I'd think something like

```
let fromPosition = leftPostion
if (reversed) {
  fromPosition = rightPosition
}
```

that way `from` is always correct. I'd prefer to make it easy to read than have fewer lines here. So maybe we also take the ternaries out of the `animate` spot and put it above with an `if` statement if that's clearer

Reply:
That sounds like a better idea 👍